openmetadata-service/src/main/java/org/openmetadata/service/events/scheduled/PipelineServiceStatusJob.java

```diff
@@ -0,0 +1,58 @@
+package org.openmetadata.service.events.scheduled;
+
+import static org.openmetadata.service.events.scheduled.PipelineServiceStatusJobHandler.JOB_CONTEXT_CLUSTER_NAME;
+import static org.openmetadata.service.events.scheduled.PipelineServiceStatusJobHandler.JOB_CONTEXT_METER_REGISTRY;
+import static org.openmetadata.service.events.scheduled.PipelineServiceStatusJobHandler.JOB_CONTEXT_PIPELINE_SERVICE_CLIENT;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import java.util.Map;
+import javax.ws.rs.core.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.openmetadata.sdk.PipelineServiceClient;
+import org.openmetadata.service.exception.PipelineServiceStatusJobException;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+
+@Slf4j
+public class PipelineServiceStatusJob implements Job {
+
+  private static final String HEALTHY_STATUS = "healthy";
+  private static final String STATUS_KEY = "status";
+  private static final String COUNTER_NAME = "pipelineServiceClientStatus.counter";
+  private static final String UNHEALTHY_TAG_NAME = "unhealthy";
+  private static final String CLUSTER_TAG_NAME = "clusterName";
+
+  @Override
+  public void execute(JobExecutionContext jobExecutionContext) {
+
+    PipelineServiceClient pipelineServiceClient =
+        (PipelineServiceClient)
+            jobExecutionContext.getJobDetail().getJobDataMap().get(JOB_CONTEXT_PIPELINE_SERVICE_CLIENT);
+    PrometheusMeterRegistry meterRegistry =
+        (PrometheusMeterRegistry) jobExecutionContext.getJobDetail().getJobDataMap().get(JOB_CONTEXT_METER_REGISTRY);
+    String clusterName = (String) jobExecutionContext.getJobDetail().getJobDataMap().get(JOB_CONTEXT_CLUSTER_NAME);
+    try {
+      registerStatusMetric(pipelineServiceClient, meterRegistry, clusterName);
+    } catch (Exception e) {
+      LOG.error("[Pipeline Service Status Job] Failed in sending metric due to", e);
+      throw new PipelineServiceStatusJobException(e);
```

Comment:
do we want to throw a specific exception here? We should just log and forget about it.
Alos in catch we should register the metric saying the scheduler is not healthy